app/boards/arm/corneish_zen/widgets/output_status.c

```diff
@@ -39,31 +38,31 @@ LV_IMG_DECLARE(USB_connected);
 static sys_slist_t widgets = SYS_SLIST_STATIC_INIT(&widgets);
 
 struct output_status_state {
-    enum zmk_endpoint selected_endpoint;
+    struct zmk_endpoint_instance selected_endpoint;
     bool active_profile_connected;
     bool active_profile_bonded;
     uint8_t active_profile_index;
```

Comment:
```suggestion
```

Looks like this is no longer used here?

Reply:
@joelspadin Did you see this review comment?

Reply:
Whoops. I missed that. Done.